src/main/kotlin/org/rust/lang/core/psi/RustTextLiteral.kt

```diff
@@ -0,0 +1,4 @@
+package org.rust.lang.core.psi
+
+interface RustTextLiteral : RustLiteral {
+}
```

Comment:
Possible properties to declare here in the future:
- `val prefix: String` - text literals can also have a prefix (`b`, `r` or `br` currently)
- `val openQuote: String` and `val closeQuote: String` - may be useful for raw strings
- `val hasPairedQuotes: Boolean` - check if the literal has both open & close delimiters